PROOF.md

```diff
@@ -60,8 +60,10 @@ Create a **Transfer** object:
 ```python
 transfer = cryptocurrency_module.Transfer()  
 transfer.amount = 100  
-transfer.seed = random.getrandbits(64)  
-transfer.to.CopyFrom(types_module.PublicKey(data=bob_keys.public_key.value))
+transfer.seed = random.getrandbits(64)
+public_key = types_module.PublicKey(data=bob_keys.public_key.value)
+caller_address = cryptocurrency_message_generator.pk_to_caller_address(public_key)
+transfer.to.data = caller_address.data
```

Comment:
Do I understand it correctly that `CopyFrom` is not working here because of `protobuf` library complaining about different types of message (despite of them being the same, but compiled in different folders)?

If so, I'd suggest to return `bytes` from `pk_to_address` function to not create a footgun.

Reply:
Exactly. Yes. Return bytes it's a good alternative.